Tests/AblyChatTests/Mocks/MockRealtimeChannel.swift

```diff
@@ -86,7 +89,7 @@ final class MockRealtimeChannel: NSObject, RealtimeChannelProtocol {
         fatalError("Not implemented")
     }
 
-    enum AttachOrDetachResult {
+    enum AttachOrDetachResult: Equatable {
```

Comment:
This conformance can be reverted per comment on `state` getter

Reply:
done